### DISCUSSION TITLE
Recent regression error when release-drafter is action is called from within a reusable action

### DISCUSSION THREAD

**Discussion by ssbarnea** [release-drafter/release-drafter#1337] [General]
While use of reusable action worked for years, I recently spotted one case where it seems not to work at all and report `Resource not accessible by integration`.

https://github.com/ansible-community/pytest-ansible/actions/runs/4699543802

The reusabed pipeline is defined at https://github.com/ansible-community/pytest-ansible/blob/main/.github/workflows/ack.yml and points to https://github.com/ansible/devtools/blob/main/.github/workflows/ack.yml

As seen both of them make use of `pull_request_target` as that is known to be required. 

I have other repositories with identical acl.yml where it works so I have no idea what might cause it. One thing that might count as different here is that this repo is in another organization than the reusable pipeline, but that should not really be causing this as they are documented to work cross-organizations.

**Comment by ssbarnea** [comment]
Apparently running across orgs is not the cause as I spotted another case where this works just fine.

**Reply by CFenner** [reply]
Did you ever solve this?
